src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -249,6 +249,9 @@ private enum FormattingState
         /// </summary>
         private enum PreprocessingState { raw, processed, error }
 
+        const int defaultConsoleWidth = 120;
+        internal const int columnsThresHold = 120;
```

Comment:
> columns**ThresHold**

`threshold` is a single word, so maybe change it to `columnsThreshold`?

Reply:
This threshold is more about using `stackalloc` or managed heap allocation, so maybe we can give it a more general name, like `StackAllocThreshold`.

As for the naming, you probably should use `StackAllocThreshold`, accoding to the naming convention [code-guidline](https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#naming-conventions)

> Use `PascalCase` to name constant local variables and fields. The only exception is for interop code where the constant should exactly match the name and value of the code you are calling via interop (e.g. `const int ERROR_SUCCESS = 0`).

Reply:
Fixed.